listenbrainz/tests/integration/test_feed_api.py

```diff
@@ -126,6 +128,80 @@ def test_it_sends_listens_for_users_that_are_being_followed(self):
         self.assertEqual(ts - 11, payload['events'][3]['created'])
         self.assertEqual('following_2', payload['events'][3]['user_name'])
 
+    def test_it_sends_listens_for_users_that_are_being_followed_new(self):
+        with open(self.path_to_data_file('valid_single.json'), 'r') as f:
+            payload = json.load(f)
+
+        ts = int(time.time())
+        # Send 3 listens for the following_user_1
+        for i in range(3):
+            payload['payload'][0]['listened_at'] = ts - i
+            response = self.send_data(payload, user=self.following_user_1)
+            self.assert200(response)
+            self.assertEqual(response.json['status'], 'ok')
+
+        # Send 3 listens with lower timestamps for following_user_2
+        for i in range(3):
+            payload['payload'][0]['listened_at'] = ts - 10 - i
+            response = self.send_data(payload, user=self.following_user_2)
+            self.assert200(response)
+            self.assertEqual(response.json['status'], 'ok')
+
+        from datetime import timedelta
```

Comment:
`listened_at` field should be in seconds not milliseconds. I think that's why the test is failing.

Reply:
Done!